snmp/metadata.csv

```diff
@@ -585,3 +585,83 @@ snmp.upsBasicStateOutputState.NoBatteriesAttached,gauge,,,,[APC UPS] The UPS sta
+snmp.jnxCosIfsetQstatMLpRedDropPktRate,gauge,,,,[Juniper EX] The rate (expressed in packets per second) at which medium-high PLP packets were RED-dropped at the output,0,snmp,
+snmp.jnxCosIfsetQstatMHpRedDropPktRate,gauge,,,,[Juniper EX] The rate (expressed in packets per second) at which medium-high PLP packets were RED-dropped at the output,0,snmp,
+snmp.jnxCosIfsetQstatHpRedDropPktRate,gauge,,,,[Juniper EX] The rate (expressed in packets per second) at which high PLP packets were RED-dropped at the output.,0,snmp,
+snmp.jnxCosIfsetQstatLpRedDropByteRate,gauge,,,,[Juniper EX] The rate (expressed in packets per second) at which medium-high PLP packets were RED-dropped at the output,0,snmp,
+snmp.jnxCosIfsetQstatMLpRedDropByteRate,gauge,,,,[Juniper EX] The rate (expressed in bytes per second) at which medium-low PLP TCP bytes were RED-dropped at the output on the given interface.,0,snmp,
+snmp.jnxCosIfsetQstatMHpRedDropByteRate,gauge,,,,[Juniper EX] The rate (expressed in bytes per second) at which medium-high PLP TCP bytes were RED-dropped at the output on the given interface.  ,0,snmp,
+snmp.jnxCosIfsetQstatHpRedDropByteRate,gauge,,,,[Juniper EX] The rate (expressed in bytes per second) at which high PLP TCP bytes were RED-dropped at the output on the given interface.,0,snmp,
+snmp.jnxCosIfsetQstatRateLimitDropPktRate,gauge,,,,[Juniper EX] The rate (expressed in bytes per second) at which packets are rate-limit dropped at the output on the given interface.,0,snmp,
+snmp.jnxCosIfsetQstatRateLimitDropByteRate,gauge,,,,[Juniper EX] The rate (expressed in bytes per second) at which bytes are rate-limit dropped at the output on the given interface.,0,snmp,
+snmp.jnxFWCounterPacketCount,count,,packet,,[Juniper EX] [Juniper MX] The number of packets being counted pertaining to the specified counter or policer.,0,snmp,
+snmp.jnxFWCounterPacketCount.rate,rate,,packet,second,[Juniper EX] [Juniper MX] The number of packets per second being counted pertaining to the specified counter or policer.,0,snmp,
+snmp.jnxFWCounterByteCount,count,,byte,,[Juniper EX] [Juniper MX] The number of bytes being counted pertaining to the specified counter,0,snmp,
+snmp.jnxFWCounterByteCount.rate,rate,,byte,second,[Juniper EX] [Juniper MX] The number of bytes per second being counted pertaining to the specified counter,0,snmp,
+snmp.jnxVirtualChassisPortInPkts,count,,packet,,[Juniper MX] Indicates the total number of packets received on the virtual-chassis port.,0,snmp,
+snmp.jnxVirtualChassisPortOutPkts,count,,packet,,[Juniper MX] Indicates the total number of packets sent from the virtual-chassis port. ,0,snmp,
+snmp.jnxVirtualChassisPortInOctets,count,,,,[Juniper MX] Indicates the total number of octets received on the virtual-chassis port.,0,snmp,
+snmp.jnxVirtualChassisPortOutOctets,count,,,,[Juniper MX] Indicates the total number of octets sent on the virtual-chassis port. ,0,snmp,
```

Comment:
```suggestion
snmp.jnxVirtualChassisPortInOctets,count,,byte,,[Juniper MX] Indicates the total number of octets received on the virtual-chassis port.,0,snmp,
snmp.jnxVirtualChassisPortOutOctets,count,,byte,,[Juniper MX] Indicates the total number of octets sent on the virtual-chassis port. ,0,snmp,
```

octets == bytes here ?

Reply:
_In theory_ `bytes != octets`. Bytes size depends on the underlying architecture while octets is always 8 bits.
Let's apply your suggestion because I suppose this makes no real difference _in practice_.